data/news/contra_costa.json

```diff
@@ -6,7 +6,7 @@
     {
       "id": "https://www.coronavirus.cchealth.org/health-orders",
       "url": "https://www.coronavirus.cchealth.org/health-orders",
-      "title": "\u200bUpdated Health Order: Requiring Reporting of COVID-19 Tests to Public Health Authorities",
```

Comment:
Data improvement: we should probably strip `\u200b` (zero-width space) and others like it when stripping whitespace from the title.

Reply:
See sfbrigade/data-covid19-sfbayarea#128